client/src/components/Feed/MessagesModal/MessageModal.js

```diff
@@ -68,7 +68,8 @@ const MessageModal = ({
       showModal();
     }
     sessionStorage.removeItem("msgModal");
-  }, [])
+    // eslint-disable-next-line no-use-before-define
```

Comment:
Sorry for not specifying, my bad.
But this is the rule we need there `// eslint-disable- next-line react-hooks/exhaustive-deps`
(Also you will need to empty the array in the next line)